parsec/core/fs/storage/workspace_storage.py

```diff
@@ -251,14 +250,21 @@ async def run(
         data_base_dir: Path,
         device: LocalDevice,
         workspace_id: EntryID,
-        cache_size: int = DEFAULT_BLOCK_CACHE_SIZE,
+        cache_size: int = DEFAULT_WORKSPACE_STORAGE_CACHE_SIZE,
         vacuum_threshold: int = DEFAULT_CHUNK_VACUUM_THRESHOLD,
```

Comment:
so this parameter is only used for `data_localdb` and not for `cache_locadb`, do we need to change it name to `data_vacuum_threshold` ?

Reply:
Right :+1:

Reply:
1ebdd468e